syncer/syncer.go

```diff
@@ -931,11 +931,18 @@ func (s *Syncer) syncDDL(tctx *tcontext.Context, queueBucket string, db *DBConn,
 			}
 		}
 		if err != nil {
+			s.jobWg.Done()
```

Comment:
not sure `s.jobWg.Done()` should be moved later than `s.execError.Set(err)`, to make sure caller will see the error after `s.jobWg.Wait()`

Reply:
I move all of it. Actually we now no need `s.execError.get()` in caller. If syncDDL error, subtask will pause and rollback checkpoint in syncer.Process